.gitignore

```diff
@@ -22,4 +22,7 @@ tags-private.json
 
 # Sonar Scanner
 *.ucfg
-.scannerwork
\ No newline at end of file
+.scannerwork
+
+# typeorm local config
+packages/api/ormconfig.json
```

Comment:
```suggestion
# typeorm local config
ormconfig.json
```

This should have it ignore the file regardless of what directory it's in. Thoughts?

Reply:
Yeah, that makes sense.